docs/UserGuide.md

```diff
@@ -277,26 +281,78 @@ Format: `unprioritze INDEX`
 
 ### Marking assignments as done : `done`
 
-Marks the specified assignment as done.
+Format: `done INDEX [MORE_INDEXES]`
 
-Format: `done INDEX`
+You can keep track of your undone assignments by marking assignments that you've completed as done.
 
-* Marks the assignment at the specified `INDEX` as done.
-* The index refers to the index number shown in the displayed assignment list.
-* The index **must be a positive integer** 1, 2, 3, …​
+You can mark **one or more** assignments as done by specifiying the assignment(s) `INDEX` as shown in your list. Here is
+an example with steps to follow:
 
-### Marking assignments as not done : `undone`
+1) To mark assignments with the name "" and "" shown in the figure below as done, you can simply enter `done 2 3` into
+the command line based on the indexes circled and labelled in the figure.
+
+DIAGRAM
+
+2) The two assignments will be marked as done and this is represented by the green ticks circled in the figure. 
 
-Marks the specified assignment as not done.
+DIAGRAM
+
+3) A message that includes the information of the assignments marked as done will be displayed in the Command Box.
+
+DIAGRAM
+
+<div markdown="block" class="alert alert-primary">
+
+ **:clipboard: Pointers to note:**<br>
+* At least one index must be present. For example, `done` without any index will not work.
+* The indexes **must be a positive integer** 1, 2, 3, …​
+* The indexes **must be found in your assignment list**
+* The indexes **must not be duplicated**. For example, `done 4 4` will not work.
+</div>
+
+
+### Marking assignments as not done : `undone`
 
 Format: `undone INDEX`
 
-**Tip:**
-Assignments are marked as not done by default.
+You can mark assignment that you have not completed as undone by specifying the assignment `INDEX` as shown in your
+ list.
```

Comment:
Maybe phrasing it as "You can mark your assignments you have marked as done as undone...." would be better? Since currently it sounds like undone assignments can be marked with a specialised undone tag instead of allowing the user to mark done assignments as undone.

Reply:
ahh okay thanks for pointing that out